tutorial/README.md

```diff
@@ -50,7 +50,7 @@ If there are errors, the import tab will remain open and display the errors and
 
 *If you know that the GTFS is valid, don't hesitate to open a new issue on [the Transition github](https://github.com/chairemobilite/transition), with a description, the error message and a link the GTFS file used.*
 
-But at this point, there should be data available in Transition!
+But at this point, there should be data available in Transition! You can see what was just imported, navigate the lines and paths, view the stop nodes, etc.
```

Comment:
Do we navigate the line or we navigate the interface and scroll the map and see the various line and path?